_includes/plugins/powerpaste_import_types.md

```diff
@@ -0,0 +1,7 @@
+The supported string-based values are:
```

Comment:
```suggestion
The supported string values are:
```

Reply:
Ha, that was string and it got switched in an earlier review. @tylerkelly13 your call?

Reply:
hmmm leave as-is pls